app.py

```diff
@@ -145,41 +176,15 @@ def _do_schedule_core_solver_jobs(
     )
 
     solvers_run = openshift.schedule_all_solvers(
-        packages=f"{package_name}=={package_version}",
-        output=output,
-        indexes=index_urls,
+        packages=f"{package_name}=={package_version}", output=output, indexes=index_urls
```

Comment:
```f"{package_name}==={package_version}"``` so it is [arbitraty match](https://www.python.org/dev/peps/pep-0440/#arbitrary-equality).